includes/class-freemius.php

```diff
@@ -275,14 +275,65 @@ private function __construct( $slug ) {
 				FS_Api::clear_cache();
 			}
 
-			$this->_register_hooks();
+			if ( $this->_caller_is_plugin() ) {
```

Comment:
@fajardoleo `_register_hooks()` hooks some methods that are relevant both for plugins and themes. Therefore, you cannot just skip it when it's a theme.

Therefore, execute that method in both cases, and wrap the hooks that related only to plugins with a relevant `if` statement.